lab-doug/lib/greet.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
```

Comment:
perfect, this is exactly how this file shold look 

functions should only try and solve a single thing at a time :)